tensorrt_llm/llmapi/mpi_session.py

```
@@ -125,24 +125,30 @@ def _abort_on_timeout(self, fut: Future, timeout: float, reason=None):
         try:
             fut.result(timeout=timeout)
         except TimeoutError:
-            logger.critical("MpiSession shutdown timeout, aborting...")
+            logger.critical(f"MpiSession shutdown timeout after {timeout}s; "
+                            "calling MPI_Abort to force-kill stuck ranks...")
             if reason is not None:
                 logger.info(f"Reason to shutdown: {repr(reason)}")
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)